index.js

```diff
@@ -28,10 +28,17 @@ function getModules() {
   eslintMod.filename = eslintLoc;
   eslintMod.paths = Module._nodeModulePaths(path.dirname(eslintLoc));
 
-  var Definition = eslintMod.require("escope/lib/definition").Definition;
-  var escope  = eslintMod.require("escope");
+  try {
```

Comment:
Yeah, my personal preference here would also be to avoid the hoisting, but I tried to stick to the style from the file in this case!